translations/messages.fr.xlf

```diff
@@ -1174,6 +1182,66 @@
         <target>Bravo ! Voyez, votre pièce n'a plus de case voisine libre après avoir récupéré la tuile, mais est restée car c&apos;était votre tour.
         Celle de l&apos;adversaire a disparu car la capture de la tuile lui a enlevé sa dernière case voisine libre !</target>
       </trans-unit>
+      <trans-unit id="3460005588993308010" datatype="html">
+        <source>You do not have any pieces of this type anymore.</source>
+        <target>Vous n'avez plus de pièces de ce type.</target>
+      </trans-unit>
+      <trans-unit id="1718016291859374582" datatype="html">
+        <source>You cannot play here: this space is already full.</source>
+        <target>Vous ne pouvez pas jouer ici : cette case est déjà pleine.</target>
+      </trans-unit>
+      <trans-unit id="3011799294333722541" datatype="html">
+        <source>You cannot add more pieces to the target space, as it will contain more than 4 pieces.</source>
```

Comment:
You cannot add more pieces to the target **stack**

Reply:
Je voulais éviter de parler de stack dans les messages/tutos, mais du coup vu ton commentaire je vais garder, stack/pile est assez clair poure personne lambda je pense ! (fait)

Reply:
Je sais pas si ça fait plus clair, dans ma tête c'est une pile va dans une case et une pièce va dans une case vide ou sur une pile. Sans suivre ça on finit par dire "déplacer une case dans la case voisine" :p